instagram-clone/src/App.js

```diff
@@ -0,0 +1,26 @@
+import React, {Component} from 'react';
+import './App.css';
+import PostsPage from './components/PostContainer/PostsPage';
+import withAuthenticate from './authentication/withAuthenticate';
+import Login from './components/Login/Login';
+
+const ComponentFromWithAuthenticate = withAuthenticate(PostsPage)(Login)
+
+class App extends Component {
```

Comment:
Since we're no longer using state in App we could make it a functional component